metadata.json

```diff
@@ -37,7 +37,8 @@
       "operatingsystem": "RedHat",
       "operatingsystemrelease": [
         "7",
-        "8"
+        "8",
+        "9"
```

Comment:
I guess this also adds CentOS 9 support? can you add that as well to the metadata.json?

Reply:
updated